core-bundle/src/Resources/contao/library/Contao/EditableDataContainerInterface.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao;
+
+interface EditableDataContainerInterface
+{
+	public function create();
+
+	public function cut();
+
+	public function copy();
+
+	public function move();
+
+	public function edit();
+}
+
+class_alias(EditableDataContainerInterface::class, 'editable');
```

Comment:
Should we also add a comment here, that we can remove this in 5.0?

Reply:
Probably not needed as we will remove all `class_alias()` calls from all Contao classes.